### ISSUE TITLE
Name?

### ISSUE THREAD

**Issue by Efreak** [pgengler/pinot#27]
Pinot is already in use by [another application](https://apps.ubuntu.com/cat/applications/pinot/) whose binary name and package name are both pinot on Ubuntu, Fedora, and probably other distros as well. [It seems nano had this problem as well](http://en.wikipedia.org/wiki/GNU_nano#History).

**Comment by pgengler** [comment]
Yeah, I didn't think too much about that when I picked the name, and only found out that there was an existing package with the name later.

Any suggestions for a different name? The first thing that jumps into my head is "noir", which doesn't seem to be used and is a little shorter.
